webapp/utils/utils.jsx

```diff
@@ -686,24 +686,19 @@ export function applyFont(fontName) {
     }
 }
 
-export function changeCss(className, classValue, classRepeat) {
-    // we need invisible container to store additional css definitions
-    var cssMainContainer = $('#css-modifier-container');
-    if (cssMainContainer.length === 0) {
-        cssMainContainer = $('<div id="css-modifier-container"></div>');
-        cssMainContainer.hide();
-        cssMainContainer.appendTo($('body'));
+export function changeCss(className, classValue) {
+    let styleEl = document.querySelector('style[data-class="' + className + '"]');
```

Comment:
So I just changed it to:
`let styleEl = document.querySelector('style[data-class="' + className + classValue + '"]');`
To get it to work.